pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -90,7 +90,10 @@ func fixKvmEnabled() error {
 func getLibvirtCapabilities() (*libvirtxml.Caps, error) {
 	stdOut, _, err := crcos.RunWithDefaultLocale("virsh", "--readonly", "--connect", "qemu:///system", "capabilities")
 	if err != nil {
-		return nil, fmt.Errorf("Failed to run 'virsh capabilities': %v", err)
```

Comment:
Perhaps record fallback was needed?

Reply:
The full log when the fallback codepath is triggered is 
```
INFO Checking if libvirt is installed             
DEBU Checking if 'virsh' is available             
DEBU 'virsh' was found in /usr/bin/virsh          
DEBU Checking 'virsh capabilities' for libvirtd/qemu availability 
DEBU Running 'virsh --readonly --connect qemu:///system capabilities2' 
DEBU Command failed: exit status 1                
DEBU stdout:                                      
DEBU stderr: error: unknown command: 'capabilities2' 
DEBU Running 'virsh --connect qemu:///session capabilities' 
DEBU Found x86_64 hypervisor with 'hvm' capabilities 
```
it's imo explicit enough regarding what's happening, but I can add a `logging.Debugf("Falling back to getting capabilities from qemu:///session")` if you think that would help.